MapboxNavigation/FeedbackViewController.swift

```diff
@@ -305,3 +326,225 @@ extension String {
         return ceil(boundingBox.height)
     }
 }
+
+public class FeedbackSubtypeViewController: FeedbackViewController {
```

Comment:
What about moving `FeedbackSubtypeViewController` into separate file? I think it'd be beneficial for readability since `FeedbackViewController.swift` is pretty large already.

Reply:
Good idea. Done.